packages/editor-ui/src/components/Sticky.vue

```diff
@@ -156,23 +192,47 @@ export default defineComponent({
 			}
 		});
 
+		const nodeBase = useNodeBase({
+			name: props.name,
+			instance: props.instance,
+			workflowObject: props.workflow,
+			isReadOnly: props.isReadOnly,
+			emit: emit as (event: string, ...args: unknown[]) => void,
+		});
+
 		return {
 			deviceSupport,
 			toast,
 			colorPopoverTrigger,
 			contextMenu,
 			forceActions,
+			...nodeBase,
 			setForceActions,
 		};
 	},
 	data() {
 		return {
 			isResizing: false,
 			isTouchActive: false,
+			inputs: [] as Array<ConnectionTypes | INodeInputConfiguration>,
+			outputs: [] as Array<ConnectionTypes | INodeOutputConfiguration>,
```

Comment:
Isn't this overridden by `...nodeBase` in setup? (contains `inputs`/`outputs` refs)

Reply:
You're right! Removed.